package.json

```diff
@@ -62,14 +62,10 @@
     "hsw-cli": "./bin/hsw-cli"
   },
   "scripts": {
-    "lint": "eslint $(cat .eslintfiles) || exit 0",
-    "lint-ci": "eslint $(cat .eslintfiles)",
-    "lint-file": "eslint",
```

Comment:
I still use `npm run lint`, have eslint installed in global node_modules. Do we need to remove this? (the `lint-ci` I understand, sure)

Reply:
I don't think scripts should reference tools which are not locally installed. If you are using a global `eslint` might as well run it directly.

Reply:
You're keeping the eslint files in the root directory though. What if eslint was listed in `peerDependencies` or something?